pandas/core/generic.py

```diff
@@ -5716,24 +5716,81 @@ def clip_upper(self, threshold, axis=None, inplace=False):
 
     def clip_lower(self, threshold, axis=None, inplace=False):
         """
-        Return copy of the input with values below given value(s) truncated.
+        Return copy of the input with values below given value(s) trimmed.
+
+        If an element value is below the threshold, the threshold value is
+        returned instead.
 
         Parameters
         ----------
         threshold : float or array_like
-        axis : int or string axis name, optional
+            Lower value(s) to which the input value(s) will be trimmed.
+        axis : {0 or 'index', 1 or 'columns'}
             Align object with threshold along the given axis.
         inplace : boolean, default False
             Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+                .. versionadded:: 0.21.0.
 
         See Also
         --------
-        clip
+        DataFrame.clip: Trim values at input threshold(s).
+        DataFrame.clip_upper: Return copy of input with values above given
+            value(s) trimmed.
+        Series.clip_lower: Return copy of the input with values below given
```

Comment:
For comprehensiveness and consistency add the Series `clip` and `clip_lower` methods as well